tests/basic/test_one_shot_keys.cpp

```diff
@@ -383,3 +383,95 @@ TEST_F(OneShot, OSLWithOsmAndAdditionalKeypress) {
     run_one_scan_loop();
     VERIFY_AND_CLEAR(driver);
 }
+
+TEST_F(OneShot, OSLWithMoAndAdditionalKeypress) {
```

Comment:
what about chaining OSL with TT, TG, LT, DF, LM, etc?

Reply:
Added some tests for TG and TO.
Currently stuck on DF. Did not find any test of this action. Is there any ?

Reply:
DF sets the default layer, rather than the normal layer state.  so you'd need to check `default_layer_state` rather than `layer_state`. 

But in this case, it may not be necessary to test for DF.

Reply:
Added tests for DF, LT and the SFT_T ModTapKey. Do you think it is enough ?